bin/zField

```diff
@@ -150,8 +150,37 @@ if __name__ == '__main__':
             dMapDict = clusters.makeDensityMap(RADeg, decDeg, galaxyCatalog, z, dz, rMaxMpc = 1.5)
             astImages.saveFITS(args.outDir+os.path.sep+"densityMap_z%s_%.3f_%.3f.fits" % (z, RADeg, decDeg),
                                dMapDict['map'], dMapDict['wcs'])
-    
+            
+    #estimate errors for density map, CS and A
+    for montecarlo1 in range(0,1):
+        print('Starting MC %i of 1000...' % (montecarlo1+1))
+        Aerr = [] ; CSerr = [] ; Mapserr=[]
+        galaxycatalognew=galaxyCatalog
```

Comment:
Please follow the style of the rest of the code - use camelCase for variable names